docroot/themes/humsci/humsci_basic/templates/page.html.twig

```diff
@@ -1,17 +1,22 @@
-{% include humsci_basic ~ '/themes/humsci/humsci_basic/templates/components/brandbar.html.twig' with { modifier_class : brand_bar_variant_classname } %}
-
-<header class="hb-masthead">
-  <section>
-  {% if page.header %}
-    {{ page.header }}
-  {% endif %}
-
-  {% if page.search %}
-    {{ page.search }}
-  {% endif %}
-  {% if page.menu %}
-    {{ page.menu }}
-  {% endif %}
+<header class="su-masthead">
+  <a href="#main-content" class="su-skiplinks ">Skip to main content</a>
```

Comment:
I think we'll want to add the translate filter here:
`{{ 'Skip to main content'|t }}`

Reply:
https://www.drupal.org/docs/8/theming/twig/filters-modifying-variables-in-twig-templates